en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,203 @@
+
+Servo endpoints were obtained by using a servo tester to determine the servo PWM pulse width to reach the max travel of each surface in each direction.
+
+### Config & Debug
+
+Access to the Pixhawk 4 Mini requires removal of the upper mount.
+While this isn't too difficult, it was a consideration for wanting to streamline debugging in the field.
+A short right angle USB micro extension was used to allow easy access to the Pixhawk 4 Mini's USB interface.
+The USB-A end of this cable was left dangling in the battery bay.
+Similarly, a JST PH to std spaced headers adapter was made, and it was also left easily accessible in the battery bay. 
+
+### Peripherals
+
+#### ExpressLRS RX
+
+A custom cable was made to connect the ExpressLRS RX to the Pixhawk 4 Mini.
+Because the Pixhawk 4 Mini has limited uarts, the RX was connected to RC input which does not have a TX pin.
+This means that the RX will only send control data to the FCU but telemtry cannot be sent to the RX from the FCU.
+Heatshrink was used to secure the dupont connector of the cable such that it cannot back out off the headers of the ExpressLRS RX.
+
+#### Caddx Vista
+
+Another custom cable was made to connect the Caddx Vista to the FCU UART and power.
+A Molex microfit was added close to the Vista so that it could be easily disconnected without needing to gain access to the Pixhawk.
+The `msp_osd` module is used to stream telemetry to the Caddx Vista which can be seen on the DJI Goggles with the "custom OSD" feature enabled.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod.jpg)
+
+#### SIK Telemetry Radio
+
+The plastic case of the SIK telemetry radio was removed to reduce weight and decrease volume of the module.
+Heatshrink was used to electrically insulate the bare board and the radio was installed between the upper and lower flight computer mounts.
+
+## Build Results & Performance
+
+Overall, this build was a success.
+Even with the added weight of the Pixhawk 4 Mini installation, the airplane balances well and has plenty of power to retain its original STOL characterisitics.
+PX4 is easily capable of stabilizing the airplane and fine tuning of the rate loops were accomplished using the fixed wing autotune module.
+The results of tuning can be found in the parameter file attached below.
+In testing I found takeoffs can be as short as only 10ft using no flaps.
+I use full flaps on landing to slow the otherwise slippery airframe.
+
+### Performance
+
+- Stall speed (no flaps): 14MPH indicated
+- Cruise speed: 35-65MPH
+- Takeoff roll (with full flaps): < 10ft
+- Endurance: ~24 minutes on 5.2Ah 4s2p LiIon, ~12 minutes on 3.6Ah 4S LiPo
+
+### Videos
+
+[Spot Landing](https://www.youtube.com/watch?v=vMFCi3G5s6E)
+
+[A Quick Flight](https://youtu.be/1DUV7QjcXrA)
+
+### Flight Logs
+
+[Evening Flight (Includes video)](https://review.px4.io/plot_app?log=d3f2c1f9-f802-48c1-ab5d-3983fc8b8719)
+
+### Parameter File
+

```

Comment:
Please add instructions or link to instructions on how these would be applied by a user who is just getting started?

Note, this is a lot of params - is it all the params or just the Diff vs the generic frame?

Reply:
I'll add a blurb on how to load them. As far as the scope of the params, this is a whole dump of the entire, I could do the airframe dump. I guess that'd probably be better to remove the calibration params.

Reply:
Yes, we should point to calibration docs instead in the basic config page and state that people should still do that for their setup before flying.